### ISSUE TITLE
Light theme port

### ISSUE THREAD

**Issue by vidjul** [ThomasAlban/everforest-zed#1]
Hey there!

Thanks for porting out Everforest into Zed 🤩

Just opening the issue out of curiosity to know if you've planned to port the light theme too? It would be lovely!

**Comment by ThomasAlban** [comment]
Yeah I'll definitely get to porting the light theme when I have time over the next few days! I just wanted to get the initial version of the theme out as fast as possible. There are a few other issues with the theme at the moment such as it not supporting multiple 'players', and I'll get to sorting those as well soon enough :)

**Comment by saligrama** [comment]
Is there any update on this? I use day/night theme switching heavily in my workflow.

**Comment by sjoeboo** [comment]
+1 would love the light theme port for this

**Comment by Dimen61** [comment]
vote for the light theme port at 2025

**Comment by kimberli** [comment]
I quickly gave a light theme a shot: https://github.com/kimberli/everforest-zed

Happy to contribute it here / publish it separately as a Zed extension
